Source/UI/FloatingButton.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FloatingButton.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 2/12/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import SwiftUI
+
+struct FloatingButton: View {
+
+    var count: Int
+
+    var isLoading: Bool
+
+    var proxy: ScrollViewProxy?
+
+    @SwiftUI.Environment(\.refresh)
+    private var refresh
```

Comment:
Whoah this is weird. TIL.

Reply:
Also, SwiftLint has been making us put property wrappers on separate lines from the var. To me this would look cleaner as `@SwiftUI.Environment(\.refresh) private var refresh` all on one line. Same goes for most of our `@Published` variables. 

How would you feel about disabling that SwiftLint rule? Not in this PR necessarily. Just in general.

Reply:
I agree. I checked the rule and it doesn't seem consistent to me. The rule has exceptions for some wrappers, like @IBOutlet so it looks very arbitrary to me. Plus, all example codes from Apple put the property wrapper in the same line.